Documentation rewrite

d2dyno opened the pull request:
Based on input from https://github.com/PAhelper/PhoenixAdult.bundle/issues/233, this PR aims to streamline a few things:

1. How quickly a user can understand the manual searching/matching process.
2. Using universal, consistent terminology and definitions.
3. Having a single metadata point, for all currently supported sites, in a non-confusing manner.

Currently, I have only started the work. For my own sake, here are the items I want to accomplish:

- [x] Finish writing Manual search usage
- [x] Write simple terms for each level of search capability, linking back to their full meaning.
- [x] For any document that gets too long, separate them into their own files (currently thinking: `sitelist.md`, `manualsearch.md` `filenaming.md`)
- [x] Rewrite Site list to match the above rewrites - Pending level 3 only
- [x] Proof-read sitelist.md for consistency and clarity
- [x] Add search types for sites with none listed so far

d2dyno commented:
@claygoldfinch any thoughts on the [manualsearch.md](https://github.com/d2dyno/PhoenixAdult.bundle/blob/master/docs/manualsearch.md) document? Going for clear definitions, as well as easy first-timer readability. Once I am confident this document is at a good starting point, I will overhaul the site list.

d2dyno commented:
I have now shortened the main readme, to provide basic information, but to ultimately link to the new docs. Please let me know if I skimmed the readme too much and some portion does not make sense.

claygoldfinch commented:
Hell yeah, this looks awesome. I'll do a more thorough review tomorrow, but super impressed so far. Nice work!

claygoldfinch commented:
I've had a chance to look over this, and I think there's a few changes I'd make:

1. Let's combine SceneID Search and SceneID match into a single term, "SceneID Match". While they are technically different, I don't think it's going to effect the average user much.
2. Likewise, let's combine Date Search and Date Match into a single term, "Date Match". Same idea as above, plus I don't know of any sites who actually support Date Search.

Another thing I want to note (for the sake of clarity), though this doesn't necessarily have to be added to the document: Adding a date to a search term will never cause an issue, as long as the date is immediately after the Site Name. If that is the case, the date will be stripped from the search term and analyzed long before the search term is plugged into it's associated site. For example (and I picked DirectURL as it's the most picky about finding a match):

"SiteName - Date - DirectURL" No issues with finding a match
"SiteName - DirectURL - Date" Will have issues matching
"SiteName - Scene Title - DirectURL" Will have issues matching
"Site Name - DirectURL - Scene Title" Will have issues matching

That said, whether or not adding a date will actually improve your search results is entirely dependent on the site.

Based on all the info above, I think this is how I would go about doing the whole "Levels":

**Level One: Enhanced Search**
- Title and/or Actor Search available
  - Date Match/SceneID Match (if a SceneID is available) will improve matching

**Level Two: Limited Search**
- Title and/or Actor Search available
  - No Date Match/SceneID Match

Level Three: Exact Matching
- SceneID Only
  - Date can be added before SceneID, but as the agent is basically just plugging the SceneID into a URL only one match will be found.
  - Title/Actor/Whatever can be added after the SceneID, but again only one scene is actually being found so it's not actually gonna "improve" matching (though the code may boost the score of the one result available)
- Direct URL Only (Let's use this name as it's similar to SceneID Only)
  - Date can be added before the URL, but (like SceneID) will not actually "improve" matching
  - Nothing can be added after the URL as it will cause issues

Also, let's set Date Add as it's own thing, outside of the Level system. Only 2-3 sites support it, and I think it's different enough that people may want to know when specific sites support it.

Sinjin69 commented:
I'm not a fan of the levels nomenclature.
I do like, Enhanced Search, Limited Search, and Exact Match

claygoldfinch commented:
Enhanced/Limited/Exact works for me! Like you said, I think the main goal is just to simplify terminology, and it does just that.

Sinjin69 commented:
I think simplifying is great. My problem is that the level numbers just feel arbitrary. I was left wondering, does Level 3 mean that I inherit Level 2 and 1, or does LLevel 1 mean the best.

I how it was referred to as Enhanced, Limited, and Exact. Hopefully that will avoid any hierarchical connotations of using numbers.

d2dyno commented:
I updated the terminology, and adjusted examples and definitions to reflect date match/sceneid match merge with search terms. I feel the [Limited Search](https://github.com/d2dyno/PhoenixAdult.bundle/blob/master/docs/manualsearch.md#limited-search) section is a bit barren, but I am not really sure anything else fits in there. I have one or two more style changes to make, but this document is ready other than that I think. Will comb through the site list next.

d2dyno commented:
Site list updated. I would like to have these resolved before moving to merge:

Missing search types:
```
#### - Karups
#### - Private
#### - SexArt
#### - SinsVR
#### - TeenMegaWorld Network
#### - Tonights Girlfriend
#### - VRBangers
```

Not sure what to categorize these as:
```
#### - Bratty Sis *SceneID (BrattySis - id) *Date Search
#### - Hot Crazy Mess *SceneID Match (HotCrazyMess - id) *Date Search
#### - NewSensations *Actress Search *DVD Search (exact spelling)
#### - NF Busty *shoot ID (NFBusty - id) *Date search
#### - Nubiles.net *Shoot ID (NubilesNet - id) *Date search
#### - Nubiles-porn.com Network *Shoot ID (Nubilesporn - id) *Date search
#### - Nubilesfilms *shoot ID (Nubilesfilms - id) *Date search
#### - Swallowed / TrueAnal / Nymphos *SceneID Only *Date Match
#### - That Sitcom Show *SceneID (ThatSitcomShow - id) *Date Search
```

d2dyno commented on docs/manualsearch.md:
```diff
@@ -0,0 +1,74 @@
+  - **Actor**
+
+## Exact Match
+#### No search available.
```
I revised this a little bit, for readability:
```
#### No search available.
*Locating the correct scene is entirely dependent on entering the correct SceneID or Direct URL. In some cases, you may add additional terms to your search. However, these will not increase the possibility of locating the correct scene and are only implemented for convenience.*
+ **SceneID**
  - Can add the the Date (before the SceneID)
  - Can typically add a Title/Actor (after the SceneID).
+ **Direct URL**
  - Can add the the Date (before the URL)
  - Adding any terms, such as Title/Actors, after the URL will cause issues with matching.

d2dyno commented on docs/sitelist.md:
```diff
@@ -0,0 +1,763 @@
+-   Teens Like It Big
+-   Teens Like It Black
+-   ZZ Series
+#### - Bratty Sis *SceneID (BrattySis - id) *Date Search
```
By title supported, does that mean it enhances the search, or is an alternate method? Just want to be sure

d2dyno commented:
Updated per comments. The only thing left on my list is the sites which are missing any search type (in the previous comment).

Also thanks @claygoldfinch for all the time to work with me to make this all clear and accurate!

d2dyno commented:
In an effort to also preserve history, but still shorten the readme, I moved all the old changes to their own file, CHANGELOG.md, which is in most official git specs.

Should the developers ever want, you can also use some neat tools to create 'versions' with this file, here is one I was looking at https://github.com/github-changelog-generator/Github-Changelog-Generator/blob/master/CHANGELOG.md

claygoldfinch commented:
**Additions:**

- MetArt and MetArtX under label "MetArt Network". Also, put already-supported sites Viv Thomas and SexArt under this label. Limited Search.
- Little Caprice - Enhanced Search
- WowGirls - Limited Search, Date Add
- VIPissy - Limited Search, Date Add
- GirlsOutWest - DirectURL
- Girls Rimming - DirectURL
- Gangbang Creampie - Enhanced Search
- DadCrush - SceneID, Date Add
- Step Secrets - Enhanced Search
- VRHush - DirectURL

claygoldfinch commented:
Move all these sites under one label, "Other Nubiles Sites". SceneID Only.
  - Anilos
  - Bratty Sis
  - Hot Crazy Mess
  - NF Busty
  - Nubilesfilms
  - NubilesNet
  - That Sitcom Show

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - SpermSwap
+  - TamedTeens
++ #### PervMom | Matching type: *[Exact](./manualsearch.md#exact-match)* - **DirectURL**
++ #### Porndoe Premium Network | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Enhanced.

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - Xchimera
+  - XXX Omas
+  - XXXShades
++ #### PornFidelity | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Enhanced.

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - XXX Omas
+  - XXXShades
++ #### PornFidelity | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### PornPros Network | Matching type: *[Exact](./manualsearch.md#exact-match)*
```
Might want to add "DirectURL"

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - ShadyPi
+  - SquirtDisgrace
+  - TeenBFF
++ #### Other PornPros sites | Matching type: *[Exact](./manualsearch.md#exact-match)*
```
Might want to add "DirectURL"

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - SpyFam
+  - Tiny4k
++ #### PrettyDirty | Matching type: *[Enhanced](./manualsearch.md#enhanced-search)*
++ #### Private
```
Limited

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
++ #### Screwbox | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### SexArt | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### SexBabesVR | Matching type: *[Exact](./manualsearch.md#exact-match)*
++ #### SinsVR
```
SceneID Only

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - PornstarTease
+  - RawAttack
+  - TheStripperExperience
++ #### StasyQ VR
```
SceneID Only

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - The Real Workout
+  - This Girl Sucks
+  - Titty Attack
++ #### TeenFidelity | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Enhanced.

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - X Angels
+  - Teen Sex Movs
++ #### That Sitcom Show | Matching type: *[Exact](./manualsearch.md#exact-match)* - **SceneID**
++ #### Tonights Girlfriend 
```
Limited (Actor Search).

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - This Girl Sucks
+  - Titty Attack
++ #### TeenFidelity | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### TeenMegaWorld Network
```
Enhanced.

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
++ #### VRBangers
++ #### VirtualRealPorn | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### VirtualTaboo | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### Vixen | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Enhanced.

claygoldfinch commented:
I think this is about set. You may want to add in a sentence or two re: Date Add, as I didn't see it covered anywhere. Basically just say "Some sites doesn't make release dates available, so the agent will strip the date from your filename/search term instead". Other than that, everything looks good!

claygoldfinch commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - Twistys Hard
+  - When Girls Play
++ #### Viv Thomas | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### VRBangers
```
Also this is out of order, alphabetically.

d2dyno replied:
Good catch again! Likely happened because `sort` will put capitals above non-capital.

d2dyno commented on docs/sitelist.md:
```diff
@@ -0,0 +1,802 @@
+  - VIP Crew
+  - We Live Together
+  - Wives in Pantyhose
++ #### SexArt
```
Great catch!

d2dyno commented:
Updated per all requests. I did notice 'ManyVids' still has no category, and it seems we skipped updating this one `NewSensations *Actress Search *DVD Search (exact spelling)`

claygoldfinch commented:
ManyVids is SceneID Only.

New Sensations is kinda in a state of disrepair (working on it), but let's call it DirectURL only.

d2dyno commented:
Ready to merge.
